### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by beckstefan**

### CODE REVIEW

File: doc/sphinx/install.rst [hunk 1]
```diff
@@ -98,14 +99,18 @@ Install and configure the search engine
...
 
 `Download Elasticsearch <https://www.elastic.co/downloads/elasticsearch>`_
 and unzip it::
 
     cd elasticsearch-<version>
     ./bin/elasticsearch    # Add -d to start elasticsearch in the background
 
+Alternatively you can install the .rpm or .deb package, see the documentation of Elasticsearch for further information.
+
+Make sure to set the initial `heapsize https://www.elastic.co/guide/en/elasticsearch/reference/5.6/heap-size.html` accordingly.
```
Comment (Phyks) [inline doc/sphinx/install.rst]: I think there are some `<` / `>` missing here.

File: doc/sphinx/install.rst [hunk 2]
```diff
@@ -119,7 +124,7 @@ Finally, create a file ``dissemin/settings/__init__.py`` with this content::
 
 Depending on your environment, you might want to override ``STATIC_ROOT`` and ``MEDIA_ROOT``, in your ``__init__.py`` file. Moreover, you have to create these locations.
 
-Don't forget to edit ``ALLOWED_HOSTS`` for production.
+Don't forget to edit ``ALLOWED_HOSTS`` for production or if your django server does not run on *localhost:8000*.
```
Comment (Phyks) [inline doc/sphinx/install.rst]: Default port is `8080` I think, not `8000`, cf https://github.com/dissemin/dissemin/blob/master/Vagrantfile#L28.

File: doc/sphinx/install.rst (hunk 1 above)
Comment (beckstefan) [reply doc/sphinx/install.rst]: Done in https://github.com/dissemin/dissemin/pull/566/commits/ec9032654f40dcd5d129c811a2e9a5b9fb31fd48

File: doc/sphinx/install.rst (hunk 2 above)
Comment (Phyks) [reply doc/sphinx/install.rst]: Super! Thanks!

**Pull request merged**